app/controllers/health_monitor/health_controller.rb

```diff
@@ -14,7 +14,7 @@ def check
       respond_to do |format|
         format.html
         format.json do
-          render json: statuses
+          render json: statuses.to_json
```

Comment:
`render json: statuses` should have worked out of the box, so I'm not sure why it didn't work for you. Are you using any special or outdated serialization gems/overrides?

Reply:
Yup, you're right. Did a quick dig and found the culprit to be the gem `active_model_serializers`. That being said, it doesn't hurt by adding that explicit conversion in, no?

Reply:
It might look weird why is it explicitly converted this way, while not in other places. Can you add `statuses.to_xml`, for symmetry?

Reply:
updated